### ISSUE TITLE
Error running Pylint Error

### ISSUE THREAD

**Issue by tobiasfielitz** [AtomLinter/linter-pylint#242]
Getting the following error in the dev console:

```
[Linter] Error running Pylint Error: Using config file /Users/ninja/.pylintrc
    at Object.<anonymous> (main.js [sm]:134)
    at Generator.next (<anonymous>)
    at step (/Users/ninja/.atom/packages/linter-pylint/lib/main.js:7)
(anonymous) @ linter-registry.js [sm]:137
```

Think I've set all paths/settings correctly:
![screen shot 2018-02-01 at 10 57 44 am](https://user-images.githubusercontent.com/9712861/35654056-3b9e7da0-073f-11e8-824f-5dc0407947e7.png)

.pylintrc is the default generated by:

`pylint --generate-rcfile > /Users/ninja/.pylintrc`

It works from the command line:

```
/Users/ninja/projects/<project>/envs/atom/env/bin/pylint log.py
Using config file /Users/ninja/.pylintrc
************* Module log
W:  9, 0: FIXME: does this flush straight away in twistd? (fixme)
C:  1, 0: Missing module docstring (missing-docstring)
C: 15, 4: Missing method docstring (missing-docstring)
C: 18, 4: Missing method docstring (missing-docstring)
C: 21, 4: Missing method docstring (missing-docstring)
C: 29, 0: Missing function docstring (missing-docstring)
```

Versions:
**linter 2.2.0**
**linter-pylint 2.1.0**
**linter-ui-default 1.6.10**

Any ideas?

**Comment by tobiasfielitz** [comment]
I have tried removing the _.pylintrc_ setting. But it still seems to be stored somewhere:

```
[Linter] Error running Pylint Error: Using config file /Users/ninja/.pylintrc
    at Object.<anonymous> (/Users/ninja/.atom/packages/linter-pylint/lib/main.js:161:17)
    at Generator.next (<anonymous>)
    at step (/Users/ninja/.atom/packages/linter-pylint/lib/main.js:7:273)
```

**Comment by ddaanet** [comment]
Fixed by #237
